apps/wakunode2/external_config.nim

```diff
@@ -83,12 +83,17 @@ type
       desc: "Private key for broadcasting transactions",
       defaultValue: "",
       name: "rln-relay-eth-private-key" }: string
-  
+
     rlnRelayUserMessageLimit* {.
       desc: "Set a user message limit for the rln membership registration. Must be a positive integer. Default is 1.",
       defaultValue: 1,
       name: "rln-relay-user-message-limit" .}: uint64
 
+    rlnEpochSizeSec* {.
```

Comment:
Please add a validator for this field - we don't want operators to shoot themselves in the foot due to memory footprint of too large epoch sizes

Reply:
you mean to limit it to a given value? which one?

Reply:
Possibly 1 day? the operators currently have no idea about the memory usage by bumping this value up

Reply:
note that an operator is expected to adhere to an existing cluster, eg cluster-id=1, but not define their own parameters. and well, if someone forks waku, they should have enough knowledge to know that 1 day is probably a bad decision.

unsure if we should enforce a limit. i mean, does it make sense to have a max number of peers of 1000000? maybe not, but we don't check for it?